cmake/hip.cmake

```diff
@@ -166,6 +174,7 @@ find_package(hiprand REQUIRED)
 find_package(hipsparse REQUIRED)
 # At the moment, for hiprand to work also rocrand is required.
 find_package(rocrand REQUIRED)
+find_package(ROCTX)
```

Comment:
Similar question here. You require NVTX, but dont do that for ROCTX. If it is not found, then are the profiler hook loggers with the respective backend just not usable, or does Ginkgo run into compiling issues.

Reply:
AMD decided to not provide a full rocm meta-package AFAIK, so the installs are more fragmented than CUDA

Reply:
I see. I have no issues with disabling it.